common/src/main/java/org/apache/celeborn/common/protocol/RpcNameConstants.java

```diff
@@ -19,18 +19,19 @@
 
 public class RpcNameConstants {
   // For Master
-  public static String MASTER_SYS = "MasterSys";
-
+  public static String MASTER_SYS = "Master";
   // Master Endpoint Name
   public static String MASTER_EP = "MasterEndpoint";
 
   // For Worker
-  public static String WORKER_SYS = "WorkerSys";
```

Comment:
What will happen if the server is using this PR and the client is old? Will this name change introduce a compatible issue?

Reply:
The xx_SYS is just used for startup logs before this pr. I think it doesn't matter.